mudita/mudita-center
# [CP-1764] Backup space validation handled

Jira: [CP-1764]

**Description**

QA notes:
 - you need full a self storage space. (memory transfer can take up to 5 hours via MTP)
 - you need a UDM version package from 👉 [PurePhone-1.6.0-RT1051-image.tar.xz](https://s3.console.aws.amazon.com/s3/object/muditacenterosreleaseweb-muditaoslatestreleases07-fomskribsxjs?region=eu-central-1&prefix=v3/production/PurePhone/latest/PurePhone-1.6.0-RT1051-image.tar.xz)

Recommendations:
- please limit yourself to code review and leave the integration check to the QA 😉 


<details>
<summary><b>Screenshots</b></summary>

<img width="1266" alt="Zrzut ekranu 2023-01-11 o 16 30 18" src="https://user-images.githubusercontent.com/17147149/211847892-86dfdf53-48a7-420c-bdea-9518dd2830d2.png">


</details>


[CP-1764]: https://appnroll.atlassian.net/browse/CP-1764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
